catroid/src/main/java/org/catrobat/catroid/web/CatrobatWebClient.kt

```diff
@@ -65,3 +105,67 @@ fun Map<String, String>.createFormEncodedRequest(url: String): Request {
         .post(formEncodingBuilder.build())
         .build()
 }
+
+private fun getPrivateKey(): PKCS8EncodedKeySpec {
+    val privateKeyInputStream: InputStream = BuildConfig.CLOUDFLARE_PRIVATE_KEY.byteInputStream()
+    val privateKeyByteArray =
+        ByteArray(privateKeyInputStream.available())
+    privateKeyInputStream.read(privateKeyByteArray)
+    val privateKeyContent = String(privateKeyByteArray, Charset.defaultCharset())
```

Comment:
Do not manually mangle around with the key. There has to be a function to import a standard X509 certificate...


==> E.g. use the Java Keystore functionality?